.circleci/config.yml

```diff
@@ -94,6 +94,19 @@ jobs:
           name: Type Checking including JS files
           command: yarn run type-check:js
 
+  'release if needed':
+    <<: *defaults
+    steps:
+      - checkout
+      - run: *install_yarn_version
+      - restore_cache: *restore_yarn_cache
+      - run: *run_yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Trigger a release if the latest commit is a release commit.
+          command: |
+            yarn release:trigger
```

Comment:
Every commit is pushed to `master`, this runs and it checks if the latest commit message starts with `chore: release v`. If so, it starts to release to NPM, otherwise it skips.